examples/mars/mars.html

```diff
@@ -0,0 +1,46 @@
+<html>
+
+<head>
+    <title>OpenGlobus - Mars planet</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../css/og.css" type="text/css" />
+</head>
+
+</html>
+
+<body>
+    <div id="globus"></div>
+    <script type="module">
+
+        'use strict';
+
+        import { Globe } from '../../src/og/Globe.js';
+        import { EmptyTerrain } from '../../src/og/terrain/EmptyTerrain.js';
+        import { XYZ } from '../../src/og/layer/XYZ.js';
+        import { quadTreeStrategyType } from '../../src/og/utils/quadTreeType.js';
+
+
+        const osm = new XYZ("OSM", {
+            'specular': [0.0003, 0.00012, 0.00001],
+            'shininess': 20,
+            'diffuse': [0.89, 0.9, 0.83],
+            'isBaseLayer': true,
+            'url': "https://astro.arcgis.com/arcgis/rest/services/OnMars/MDIM/MapServer/tile/{z}/{y}/{x}?blankTile=false",
+            'visibility': true,
+            'attribution': 'Data @ USGS Astrogeology Science Center, NASA, JPL, Esri'
+        });
+
+        window.globe = new Globe({
+            'name': "mars",
+            'quadTreeStrategyPrototype': quadTreeStrategyType.mars,
+            'target': "globus",
+            'terrain': new EmptyTerrain(),
+            'layers': [osm],
+            "sun": {
+                "active": false
+            }
+        });
```

Comment:
As this is a large refactor, we also need make sure all existing examples will not get break?

Reply:
Of course.